src/tile.cpp

```diff
@@ -1405,30 +1393,30 @@ void Tile::postRemoveNotification(Thing* thing, const Cylinder* newParent, int32
 		onUpdateTile(spectators);
 	}
 
-	for (Creature* spectator : spectators) {
-		assert(dynamic_cast<Player*>(spectator) != nullptr);
-		static_cast<Player*>(spectator)->postRemoveNotification(thing, newParent, index, LINK_NEAR);
+	for (auto spectator : spectators) {
```

Comment:
Could this be `const auto&`
```suggestion
	for (const auto& spectator : spectators) {
```